packages/react-admin/src/mui/field/ReferenceField.js

```diff
@@ -135,6 +145,7 @@ ReferenceField.propTypes = {
     allowEmpty: PropTypes.bool.isRequired,
     basePath: PropTypes.string.isRequired,
     children: PropTypes.element.isRequired,
+    classes: PropTypes.object,
```

Comment:
add to default props, too (`{}`) to ease tests.

Reply:
Done